topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+   link: "https://learning.skills4eosc.eu/course/view.php?id=19"
+lang: "en"
+level: "Introductory"
+extra:
+  description: "Short how to guide that guides you through the stages of the FAIR-by-Design methodology without any specific choice on tools and formats."
+  target_audience: "Instructional designers"
+  URL_resource_type: "Permanent URL"
+  learning_resource_type: "self-paced microlearning unit"
+  access_cost: "No"
+  attribution: "This microlearning unit is based on Filiposka, Sonja, Mishev, Anastas. (2024). FAIR-by-Design Microlearning (1.0.0). https://doi.org/10.5281/zenodo.11548062"
+---
+
+
+Welcome to the FAIR-by-Design Methodology Microlearning GTN adapted tutorial.
+
+> <agenda-title></agenda-title>
+>
+>Below are the main stages of the FAIR-by-Design Methodology {% cite filiposka_2023_8305540 %} that will help guide you on your journey of creating FAIR learning materials.
+>
+>![Visual representation of the FAIR-by-Design Methodology flow with its separate stages](../../images/methodology.png)
+>
+>The FAIR-by-Design Methodology is defined in [a number of stages](#main-stages) that help you incorporate the FAIR principles into the backward learning instructional design process so that the final output are FAIR learning materials from both learners and instructors perspective:
+>
+>1. [Prepare](#stage-1---prepare)
+>	- Do you understand FAIR and its implications
+>	- Define purpose, learning objectives, target audience
+>2. [Discover](#stage-2---discover)
+>	- Find existing resources
+>	- Identify potential for reuse
+>3. [Design](#stage-3---design)
+>	- Define the syllabus and structure
+>	- Identify granularity
+>	- Define facilitation materials
+>4. [Produce](#stage-4---produce)
+>	- Develop content using common apps and file formats
+>	- Define machine readable metadata
+>	- Perform internal QA check
+>5. [Publish](#stage-5---publish)
+>	- Define license and other related info
+>	- Release for instructors and learners
+>	- Enable feedback gathering
+>6. [Verify](#stage-6---verify)
+>	- Final QA check
+>	- Use gathered feedback for continuous improvement
+>7. [Continuous Improvement](#stage-7---continuous-improvement)
+>	- Create a list of potential improvements
+>	- Choose a set of improvements to be implemented
+>	- Start a new release cycle
+>
+>Review each stage and the essential steps that it includes.
+>
+>Essential information about the original microlearning unit that this adaptation is based on can be found in the [About](#about) section.
+>
+{: .agenda}
+
+If you have any questions at any stage on your journey do not hesitate to <a href="mailto:sonja.filiposka@finki.ukim.mk">contact the FAIR-by-Design methodology team</a>.
+
+May all your materials be FAIR!
+
+
+# Main stages
+
+## Stage 1 - Prepare
+
+> Before anything else, preparation is the key to success!
+{: .quote author="Alexander Graham Bell"}
+
+### First things first: What is FAIR?
+
+
+> <solution-title>Findable</solution-title>
```

Comment:
The solution box is the only option to provide answers to questions (in this case about what is FAIR) in a collapsable box. If there is a way to customise the box so we can change the title,  please let us know.

Reply:
To stay in line with the GTN formatting and style guides, could you either:
1. wrap the 4 solution boxes in a big question box with the question "What is FAIR?"
2. Just put everything in plain text, since it feels more like a rhetorical question rather than something you expect learners to really go out and find the answers themselves